main.css

```diff
@@ -108,3 +121,2555 @@ body {
       font-size: 0.875rem; /* 14px */
       font-weight: 600;
     }
+
+.prog-bar{}
```

Comment:
Also, please give this padding:
`padding: 0 2rem;`
(This ties in to other comments about removing a class from the "New Session" button)